packages/@aws-cdk/aws-glue/test/integ.table.ts

```diff
@@ -84,15 +88,22 @@ const encryptedTable = new glue.Table(stack, 'MyEncryptedTable', {
   partitionKeys,
   dataFormat: glue.DataFormat.JSON,
   encryption: glue.TableEncryption.KMS,
-  encryptionKey: new kms.Key(stack, 'MyKey'),
+  encryptionKey: new kms.Key(stack, 'MyKey', {
+    removalPolicy: cdk.RemovalPolicy.DESTROY,
```

Comment:
good catch...we should really add a linter rule for these removal policies

Reply:
I wouldn't need it after performing an integ test. I've also noticed that lambda retention periods for logs are infinite, which are not suitable in my opinion, as they can pile up upon multiple integration tests.